File: 151001/Aleksandrov/RESTLab1/src/main/java/entities/Editor.java
package entities;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Editor {
    Long id;
    String login;
    String password;
    String firstname;
    String lastname;
}

File: 151004/Ivanenko/DC_Lab3/DC_Lab2/Models/Editor.cs
using System;
using System.Collections.Generic;

namespace DC_Lab1.Models;

public partial class Editor
{
    public int Id { get; set; }

    public string? Login { get; set; }

    public string? Password { get; set; }

    public string? Firstname { get; set; }

    public string? Lastname { get; set; }

    public virtual ICollection<Tweet> Tweets { get; set; } = new List<Tweet>();
}
